pallets/asset/src/lib.rs

```diff
@@ -426,6 +436,13 @@ decl_storage! {
                 };
                 ClassicTickers::insert(&import.ticker, classic_ticker);
             }
+
+            let sys_did = &config.reserved_country_currency_codes[0].0;
+            for code in &config.reserved_country_currency_codes[0].1 { 
```

Comment:
This would only process the codes of the first identity in the vector. We need to iterate over all pairs in the vector instead.